src/h5web/visualizations/heatmap/HeatmapToolbar.tsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react';
-import shallow from 'zustand/shallow';
 import styles from './HeatmapToolbar.module.css';
 import ColorMapSelector from './ColorMapSelector';
 import Toggler from './Toggler';
-import { useHeatmapStore } from './store';
+import { useHeatmapConfig } from './config';
 
 function HeatmapToolbar(): JSX.Element {
-  const [
+  const {
     hasLogScale,
     toggleLogScale,
     keepAspectRatio,
     toggleAspectRatio,
-  ] = useHeatmapStore(state => [
-    state.hasLogScale,
-    state.toggleLogScale,
-    state.keepAspectRatio,
-    state.toggleAspectRatio,
-    shallow,
-  ]);
+  } = useHeatmapConfig();
```

Comment:
We can just get the entire config now.

Reply:
Oh sorry, I changed this back. It's a little strange to have some components that are controlled (i.e. the togglers) and some that control themselves in the toolbar, but I didn't like any of the alternatives, so I left it as is for now.